spec/car_spec.rb

```diff
@@ -37,4 +37,17 @@
 
     expect(car.total_cost).to eq(54000)
   end
+
+  it 'has not been painted' do
+    car = Car.new("Ford Mustang", 1500, 36)
+
+    expect(car.color).to eq(nil)
+  end
+
+  it 'has been painted blue' do
+    car = Car.new("Ford Mustang", 1500, 36)
+
+    car.paint!(:blue)
+    expect(car.color).to eq(:blue)
```

Comment:
This only passes because `:blue` is the same as in your `Car.paint!` method -- try changing this test to `:red` instead, and you'll see what I mean.